osd_whats_new/osd-whats-new.adoc

```diff
@@ -17,15 +17,22 @@ With its foundation in Kubernetes, {product-title} is a complete {OCP} cluster p
 
 [id="osd-q3-2024_{context}"]
 === Q3 2024
+
+* **Expanded support for GCP standard instance types.** {product-title} on {GCP} now supports standard instance types for control plane and infrastructure nodes.
```

Comment:
I'm not seeing exactly the location where QE ack'ed this exact text. Would you have a link?

Reply:
Hello @dfitzmau, 
    Thanks for reviewing. Please see PR which includes a QE review/approval for this feature: https://github.com/openshift/openshift-docs/pull/75455 (specifically this comment which has the lgtm https://github.com/openshift/openshift-docs/pull/75455#issuecomment-2151720950) 
Release notes are a recap of a feature that has been approved/published. In this example the release note points out that we have expanded standard instance types for control plane and infrastructure nodes (previously we only supported standard instance types for worker nodes.)   See original feature ticket description below (in image) . 
![image](https://github.com/openshift/openshift-docs/assets/122639474/681a133d-a052-4ce7-8476-f33fa6ae7709)